ppqm/chembridge.py

```diff
@@ -450,29 +450,55 @@ def get_canonical_smiles(smiles: str) -> str:
 
 
 def get_center_of_mass(
-    atomic_masses: Union[List[float], np.ndarray], coordinates: np.ndarray
+    atomic_masses: np.ndarray, coordinates: np.ndarray
 ) -> np.ndarray:
-    """Calculate center of mass"""
+    """Calculate the center of mass
+    
+    Args:
+        atomic_masses (np.ndarray): 
+            numpy array of atomic masses. Must have shape (n_atoms)
+        coordinates (np.ndarray):
+            numpy array of coordinates. Must have shape (n_atoms, 3)
+
+    returns:
+        The center of mass of the molecule, as a numpy array of shape (3)
+    """
 
+    assert len(atomic_masses.shape) == 1, "Input array has wrong shape" 
```

Comment:
We shouldn't use assert in function, but raise exceptions. Otherwise it is harder for others to use the code

Reply:
addressed in commit de1757a.